apps/acceptance/homescreen/regions/homescreen.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+var View = require('../../lib/view');
+var Marionette = require('marionette-client');
+
+
+function Homescreen(client) {
+    View.call(this, client);
+}
+
+Homescreen.prototype = Object.create(View.prototype);
+Homescreen.prototype.constructor = Homescreen;
+
+
+Homescreen.prototype.launch = function(){
```

Comment:
I'm not sure we need a launch method for the homescreen, as this app is always rebooted when we need it.